### PR TITLE
updated german bios

### PR DESCRIPTION
**Pull request by matheharry**

### CODE REVIEW

File: content/shared/translations/bios_de.yaml [hunk 1]
```diff
@@ -379,7 +379,7 @@ conway:
   born: 1937
   died: 2020
   bio: |
-    __John Horton Conway__ (geb. 1937) ist ein britischer Mathematiker und derzeit
+    __John Horton Conway__ (1937 - 2020) ist ein britischer Mathematiker und derzeit
```
Comment (plegner) [inline content/shared/translations/bios_de.yaml]: John Conway is still alive – we used to have "2020" as the "died" property for all currently living mathematicians, but that is no longer needed in the latest version.

File: content/shared/translations/bios_de.yaml [hunk 2]
```diff
@@ -951,17 +951,17 @@ johnson:
   died: 2020
   name: Johnson
   bio: |
-    __Katherine Johnson__ (born 1918) is an African-American mathematician.
-    While working at NASA, Johnson calculated the orbits taken by American
-    astronauts – including Alan Shepard, the first American in space, the Apollo
-    Moon landing program, and even the Space Shuttle.
+    __Katherine Johnson__ (1918 - 2020) ist eine afroamerikanische Mathematikerin.
```
Comment (plegner) [inline content/shared/translations/bios_de.yaml]: Replace "ist" with "war"

File: content/shared/translations/bios_de.yaml [hunk 3]
```diff
@@ -1402,11 +1403,11 @@ penrose:
   born: 1931
   died: 2020
   bio: |
-    __Sir Roger Penrose__ (born 1931) is a British mathematician and physicist
-    who is known for his groundbreaking work in general relativity and cosmology
-    – often collaborating with other famous scientists like Stephen Hawking and
-    Michael Atiyah. He also discovered _Penrose Tilings_: self-similar,
-    non-periodic tessellations.
+    __Sir Roger Penrose__ (1931 - 2020) ist ein britischer Mathematiker und Physiker
```
Comment (plegner) [inline content/shared/translations/bios_de.yaml]: Still alive

File: content/shared/translations/bios_de.yaml [hunk 4]
```diff
@@ -1750,7 +1751,7 @@ wiles:
   died: 2020
   name: Wiles
   bio: |
-    Der britische Mathematiker Sir __Andrew Wiles__ (geb. 1953) ist vor allem für den
+    Der britische Mathematiker Sir __Andrew Wiles__ (1953 - 2020) ist vor allem für den
```
Comment (plegner) [inline content/shared/translations/bios_de.yaml]: Still alive

File: content/shared/translations/bios_de.yaml (hunk 1 above)
Comment (matheharry) [reply content/shared/translations/bios_de.yaml]: 👍

**Comment by plegner** [comment]
Thanks for fixing this 😄

**Pull request merged**